_apps/rtd-bot.md

```diff
@@ -0,0 +1,27 @@
+---
+# A human-friendly name of your listing
+title: Read The Docs
+# A short description of what your app does
+description: Assists reviewing using Read The Docs
+# The slug of your hosted app on GitHub (https://github.com/apps/YOUR-SLUG)
+slug: rtd-bot
```

Comment:
We banned the word `bot` from being included in slugs since when your app is displayed on GitHub, it is shown next to `[bot]`. Would you consider renaming your app?

Reply:
OK I've renamed my app, and updated slug in this .md file.